hack/k8s/Bootstrap.yaml

```diff
@@ -111,9 +96,10 @@ metadata:
   namespace: spike
   name: spiffe-bootstrap-role
 rules:
-- apiGroups: ["batch"]
-  resources: ["jobs"]
-  verbs: ["get", "list"]
+# Allow Bootstrap to read/write ConfigMaps
+- apiGroups: [""]
+  resources: ["configmaps"]
+  verbs: ["get", "create", "update", "patch"]
```

Comment:
should be able to lock it down slightly more like:
```
- apiGroups: [""]
  resources: ["configmaps"]
  verbs: ["create"]
- apiGroups: [""]
  resources: ["configmaps"]
  verbs: ["get", "update", "patch"]
  resourceNames: ["spike-bootstrap-state"]
```

or, if we change api's slightly, can lock down the create too.